game.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Could you please fix Rubocop errors before merging, Since it's our last card in the project